query-service/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ dependencies {
 }
 
 application {
-  mainClassName = "org.hypertrace.core.serviceframework.PlatformServiceLauncher"
+  mainClass.set("org.hypertrace.core.serviceframework.PlatformServiceLauncher")
```

Comment:
interesting

Reply:
this was just a deprecation that I noticed, the old way would have also worked. In general, gradle is moving to use properties (which support lazy resolution) vs values. Most of our own plugins already use the property syntax.